### PR TITLE
Add a SystemService#changed_value?

### PR DESCRIPTION
**Pull request by imobachgs**
* Add changed_value?
* Alias SystemService#active? to SystemService#active.

### CODE REVIEW

File: library/system/src/lib/yast2/system_service.rb [hunk 1]
```diff
@@ -128,19 +128,24 @@ def start_mode=(mode)
...
+    # Determine whether the service will be active after calling #save
+    #
+    # @return [Boolean] true if the service must be active; false otherwise
     def active
-      return changes[:active] unless changes[:active].nil?
+      return new_value_for(:active) if changed_value?(:active)
       service.active?
     end
 
+    alias_method :active?, :active
```
Comment (joseivanlopez) [inline library/system/src/lib/yast2/system_service.rb]: Why not `active?` directly?

File: library/system/src/lib/yast2/system_service.rb [hunk 2]
```diff
@@ -128,19 +128,24 @@ def start_mode=(mode)
     # The given value will be applied after calling #save.
     #
     # @param value [Boolean] true to set this service as active
-    def active=(active)
-      if active == service.active?
-        changes.delete(:active)
+    def active=(value)
+      if value == service.active?
+        unregister_change(:start_mode)
```
Comment (joseivanlopez) [inline library/system/src/lib/yast2/system_service.rb]: :start_mode? I guess it should be `unregister_change(:active)`

Comment (joseivanlopez) [reply library/system/src/lib/yast2/system_service.rb]: I would add a test for `#active=` and `#active=`.

File: library/system/src/lib/yast2/system_service.rb (hunk 1 above)
Comment (imobachgs) [reply library/system/src/lib/yast2/system_service.rb]: It is a `leftover` (I kept them for compatibility reasons). But not needed anymore, so I've removed it.

**Comment by imobachgs** [comment]
Having those `#start`, `#stop` methods makes sense. I went for `active=` simply because it was more convenient while adapting the `ServicesManagerService`.

File: library/system/test/yast2/system_service_test.rb [hunk 3]
```diff
@@ -162,69 +162,129 @@
...
+
+      it "ignores the change" do
+        system_service.start
+        expect(system_service.changed?).to eq(false)
+      end
+    end
 
-      it "sets #active to true" do
-        expect { system_service.active = true }.to change { system_service.active? }
+    context "and the services was set to be stopped" do
```
Comment (joseivanlopez) [inline library/system/test/yast2/system_service_test.rb]: NP: service, in singular.

File: library/system/test/yast2/system_service_test.rb [hunk 4]
```diff
@@ -162,69 +162,129 @@
...
+        expect(system_service.active?).to eq(true)
+      end
+    end
+
+    context "when service was set to be stopped" do
+      before do
+        system_service.stop
+      end
+
+      it "returns true" do
```
Comment (joseivanlopez) [inline library/system/test/yast2/system_service_test.rb]: Should be "returns false".

**Pull request merged**